resources/views/admin/registrations.blade.php

```diff
@@ -5,7 +5,7 @@
 <div class="row justify-content-center">
         <div class="col-md-8">
             <div class="card">
-                <div class="card-header">@lang('admin.handle_registrations')</div>
+                <div class="card-header text-white bg-dark">@lang('admin.handle_registrations')</div>
```

Comment:
Shouldn't we override the `card-header` instead? In that way we wouldn't need to modify each and every `card-header` one-by-one.

Reply:
I only added the tag to the admin headers. So something like card-admin-header class would be the best option, but that would be only easy and nice with LESS/SASS/etc, otherwise copy paste. Will create an issue for this after the release.